tensorflow/python/lib/io/file_io_test.py

```diff
@@ -451,6 +452,12 @@ def testReadlines(self):
     lines = f.readlines()
     self.assertSequenceEqual(lines, data)
 
+  def testUTF8StringPath(self):
+    file_path = os.path.join(self._base_dir, "UTF8测试_file")
```

Comment:
It's by purpose, to test the file path with utf-8 encoding, to verify the bug fix.